src/main/java/org/terasology/magicalstats/system/MagicalStatsSystemCommands.java

```diff
@@ -23,14 +23,14 @@
 import org.terasology.entitySystem.systems.RegisterSystem;
 import org.terasology.logic.console.commandSystem.annotations.Command;
 import org.terasology.logic.console.commandSystem.annotations.CommandParam;
+import org.terasology.logic.console.commandSystem.annotations.Sender;
+import org.terasology.logic.health.EngineDamageTypes;
 import org.terasology.logic.permission.PermissionManager;
 import org.terasology.logic.players.LocalPlayer;
 import org.terasology.magicalstats.component.MagicalStatsComponent;
-import org.terasology.magicalstats.event.OnArcaneResistanceChangedEvent;
-import org.terasology.magicalstats.event.OnIntelligenceChangedEvent;
-import org.terasology.magicalstats.event.OnMagicalStatChangedEvent;
-import org.terasology.magicalstats.event.OnWillpowerChangedEvent;
-import org.terasology.magicalstats.event.OnWisdomChangedEvent;
+import org.terasology.magicalstats.component.ManaComponent;
+import org.terasology.magicalstats.event.*;
```

Comment:
Avoid wildcard imports like this (another good practice). There is a setting in IntelliJ you can increase to avoid wildcards unless there is some absurd (9999) classes from the same package. Then the IDE will happily help keep your import blocks tidy.